lib/ui/code_widget.dart

```diff
@@ -394,6 +403,10 @@ class _CodeWidgetState extends State<CodeWidget> {
   }
 
   void _onDeletePressed(_) async {
+    bool _isAuthSuccessful = await requestAuthentication("Authorize to shoq QR");
```

Comment:
Ditto

Reply:
I added the string to all the arb files except app_he.arb since he locale is not getting used.

Reply:
FYI: You just need to add entries to `app_en.arb`, community usually contributes the rest via [Crowdin](https://crowdin.com/project/ente-authenticator-app), but thanks for adding these!